README.md

```diff
@@ -250,59 +326,76 @@ Request Body:
   
 }
 ```
-18. Create Chef Profile
-Description: Allows a user to create a chef profile to showcase their culinary offerings.
-Endpoint: /chef/
-Method: POST
+25. Client Cancel Order
+Description: Allows a client to cancel an order they previously placed.
+Endpoint: /orders/cancel-order/:id
+Method: DELETE
+Parameters:
+id (required): The ID of the order to cancel.
+
+26. Get Chef Orders
+Description: Retrieves a list of orders received by a specific chef.
+Endpoint: /orders/chef-orders
+Method: GET
+
+28. Chef Update Order Status
+Description: Allows a chef to update the status of an order they are handling.
+Endpoint: /orders/chef-update-order-status
+Method: PUT
 Request Body:
 ```json
 {
-  {
-    "restaurant": "",
-    "location": "",
-    "openingHours": "",
-    "closingHours": "",
-    "contactNumber": "",
-    "description": ""
-}
+  "status": ""
 }
 ```
 
 
+
 ## Examples
 
 Here are some examples of how you can interact with the backend API using various endpoints:
 
-Example 1: Creating a new normal user account
+1. Example: Creating a new normal user account
```

Comment:
This is not a very good example, if you want to give an example of creating a chef, you should guide the user through this, first of all he/she should create a user profile, and then if he is a chef then he/she has to fille the chef info, and then he can create dishes.